### DISCUSSION TITLE
Lekérdezések, FAD, megrendelésszámok

### DISCUSSION THREAD

**Discussion by e-Officium** [nav-gov-hu/eVAT#17] [Kérdések és válaszok - Gépi interfész] [answered]
Üdvözlöm!
Az e-Officium szakembereivel áttekintettük a közzétett eáfa koncepciót, és kíváncsian várjuk a készülő fejlesztést. 

Az alábbi kérdések fogalmazódtak meg a kinyerhető adatokkal kapcsolatosan: 
1) Lesz lehetőség a NAV Adattóban elérhető adatok szűrhető lekérdezésére? Illetve akár egy adatállományba is lekérdezhetőek lesznek például egy naptári hónaphoz tartozó áfa listák? Vannak olyan ügyfeleink, akik 50 pénztárgépet üzemeltetnek, továbbá olyanok, akik több ezer számlát állítanak ki havi szinten, így gyakran 73.000 sor feletti a teljes áfa lista, ha minden pénztárgépes adatszolgáltatás külön soron jelenik meg.
2) Fordított áfa tartalmú számlák: várható, hogy módosításra kerül az ÁFA törvény, és a VTSZ szám illetve a kg adatok a számla kötelező tartalmi elemei lesznek mezőgazdasági termények és vas-és acéltermékek esetén? Jelen esetben ügyfeleink és partnereik számlázó programjából bekerülnek ezek az adatok a NAV adatszolgáltatásba, a nav online felületen látszanak az adatok, viszont onnan exportálni nem tudjuk ezeket az információkat a könyvelő programba. Jelenleg manuálisan, soronként visszük be a VTSZ számokat és a kg adatokat a könyvelőprogramba, ahonnan az áfa bevallás 07 és 08-as lapjainak adatait exportáljuk. Hatalmas segítség lenne, ha ezek az adatok is kinyerhetőek lehetnének a NAV adattóból.
3) Kiállított számlákon feltüntetett megrendelésszámok: hasonló a problémakör, mint a fordított áfás számlák esetén. A nav online adatexport nem tartalmazza a megrendelésszámokat, ami megkönnyíti a számla és a pénzügyi rendezés egyeztetését. Volna rá lehetőség, hogy az adattóból olyan adatbázist készíthessenek a könyvelők, amely tartalmazza a megrendelésszámokat is? 

Megtisztelő válaszát előre is köszönöm!
Üdvözlettel:
az e-Officium csapata

**Comment by NTCA-tax** [comment, marked as answer]
Az adattónak az elve, hogy az adatok összesítés nélkül találhatóak meg benne. Tehát ha több ezer számlát tartalmaz egy ügyfélnél egy hónap, akkor ezeket tranzakciós szinten lehet lekérdezni. Bármifajta összegzést, további feldolgozást a könyvelő programban lehet majd megtenni. Adóhivatali oldalon semmit sem fogunk összevonni ezeknél az adatköröknél. A szűréseket tervezünk, de még nem tartunk ott, hogy ezeket specifikáljuk. Jelenleg az adatköröket, adattartalmat mérjük fel, amelyekből már látszik mire lesz értelme szűrést adni az API-ban.

A VTSZ adatok kötelezőségét nem várhatóan az áfa törvény nem fogja bevezetni, mivel az irányelvvel ellentétes lenne. A számlán ettől fel lehet tüntetni (sőt fordított adózás esetén nagyon is célszerű), és az adatszolgáltatásban is célszerű szerepeltetni. Amennyiben az API-n keresztül kerül az adatszolgáltatás letöltésre, akkor ezek az adatok benne vannak az XML-ben. A felületen nem jelenítünk meg minden adatszolgáltatás adatot.  Ugyan ez vonatkozik a megrendelésszámra is. Célszerű az API irányába fejleszteni a könyvelő programot, vagy a fejlesztővel felvenni a kapcsolatot. Az adattónak nem tervezünk felületet, kizárólag adatkapcsolaton keresztül fogja a könyvelő program megkapni az adatokat.

**Comment by Amy-WYP** [comment]
收到啦~
